cmd/nerdctl/image_convert.go

```diff
@@ -66,6 +67,10 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Int("estargz-compression-level", gzip.BestCompression, "eStargz compression level")
 	imageConvertCommand.Flags().Int("estargz-chunk-size", 0, "eStargz chunk size")
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
+	imageConvertCommand.Flags().String("zstdchunked-record-in", "", "Read 'ctr-remote optimize --record-out=<FILE>' record file (EXPERIMENTAL)")
```

Comment:
Record-in should be experimental https://github.com/containerd/nerdctl/blob/main/docs/experimental.md